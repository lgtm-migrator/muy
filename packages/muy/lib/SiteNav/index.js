import React from "react"
import Nav from "../Nav"

const SiteNav = (props) => (
  <Nav
    itemScope
    itemType="https://schema.org/SiteNavigationElement"
    {...props}
  />
)

export default SiteNav
