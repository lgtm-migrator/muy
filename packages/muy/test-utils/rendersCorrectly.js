import renderer from "react-test-renderer"

const rendersCorrectly = component => {
  const tree = renderer.create(component).toJSON()
  return expect(tree).toMatchSnapshot()
}

export default rendersCorrectly
