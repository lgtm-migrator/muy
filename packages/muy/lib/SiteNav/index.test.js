import React from "react"
import rendersCorrectly from "../../test-utils/rendersCorrectly"
import SiteNav from "."

describe("<SiteNav />", () => {
  it("should render correctly", () => {
    rendersCorrectly(<SiteNav />)
  })
})
