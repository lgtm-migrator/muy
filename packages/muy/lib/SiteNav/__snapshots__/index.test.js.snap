// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SiteNav /> should render correctly 1`] = `
<nav
  className="MuiBox-root MuiBox-root-2"
  itemScope={true}
  itemType="https://schema.org/SiteNavigationElement"
  role="navigation"
/>
`;
